### PR TITLE
Bump RLS and add `autoRebuild` setting

### PR DESCRIPTION
**Pull request by alex35mil**
- [x] Bump RLS to 1.4.1
- [x] Fix formatting for `ols`
- [x] Add `autocompleteResultsFirst` setting.
- [ ] Add `autoRebuild` setting. But it doesn't seem to work ([issue](https://github.com/jaredly/reason-language-server/issues/222))

### CODE REVIEW

**Comment by alex35mil** [comment]
I also added `autocompleteResultsFirst` setting for folks who want their snippets to be on top of autocompletion list.

File: lib/config.js [hunk 1]
```diff
@@ -51,6 +51,13 @@ exports.default = {
                 order: 30,
                 default: 80,
             },
+            autoRebuild: {
+                type: 'boolean',
+                title: 'Auto rebuild on save',
+                description: 'Auto-run bsb / dune on file save',
```
Comment (zaaack) [inline lib/config.js:57]: do you mean done？

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.2.0
+* Bump `reason-language-server` to `1.4.1`.
+* Add `autoRebuild` option to settings.
```
Comment (zaaack) [inline CHANGELOG.md:3]: autocompleteResultsFirst is a good feature,   I think it worth to appear in changelog. : )

**Review by zaaack — APPROVED** [review]

**Comment by zaaack** [comment]
Only find some typo so I fixed it myself. A new version is just published `ide-reason@v1.1.3`, now enjoy!

File: lib/config.js (hunk 1 above)
Comment (alex35mil) [reply lib/config.js:57]: Haha nope, [`dune`](https://github.com/ocaml/dune)

File: CHANGELOG.md (hunk 2 above)
Comment (alex35mil) [reply CHANGELOG.md:3]: Oh, forgot to add, thanks for catching this one!

**Pull request merged**